rules/missing_punctuation.yml

```diff
@@ -0,0 +1,14 @@
+id: missing_punctuation
```

Comment:
We could also consider rules for when doc comments are necessary. Currently all modules, structs, struct fields, methods (except `new()`), and functions have doc comments. 

Note that this rule might not be ideal since we might have very small functions and test utility functions that we do not think need doc comments.

Reply:
Added to task.